src/main/java/org/example/services/AuthService.java

```diff
@@ -0,0 +1,41 @@
+package org.example.services;
+
+import io.jsonwebtoken.Jwts;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import org.example.daos.interfaces.IAuthDao;
+import org.example.exception.AuthDaoException;
+import org.example.exception.InvalidCredentialsException;
+import org.example.models.LoginRequest;
+import org.example.models.User;
+
+import java.security.Key;
+import java.util.Date;
+
+@Getter @Setter @AllArgsConstructor
+public class AuthService {
+    private final IAuthDao authDao;
+    private final Key key;
+
+    public String login(final LoginRequest loginRequest) throws
+            InvalidCredentialsException, AuthDaoException {
```

Comment:
can we put these exceptions up on line 21?

Reply:
This would lead to linter error